addons/mail/static/src/core/common/chat_hub.js

```diff
@@ -31,8 +31,15 @@ export class ChatHub extends Component {
         });
         this.options = useDropdownState();
         this.more = useDropdownState();
-        this.compactRef = useRef("compact");
-        this.compactPosition = useState({ left: "auto", top: "auto" });
+        this.ref = useRef("bubbles");
+        this.position = useState({
+            dragged: false,
+            isDragging: false,
+            top: "unset",
+            left: "unset",
+            bottom: `${this.chatHub.BUBBLE_OUTER}px;`,
+            right: `${this.chatHub.BUBBLE_OUTER + this.chatHub.BUBBLE_START}px;`,
+        });
```

Comment:
I feel that all this could probably be internal state of `useMovable()` hook, and when defining the `useMovable` we can set default position.

This is ok for now, we can do in follow-up PR.